src/bindings/python/tests/test_frontend/test_frontend_extension.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from openvino.frontend import FrontEndManager
+
+mock_available = True
+try:
+    from openvino.pybind_mock_frontend import FrontEndWrapperPaddle, FrontEndWrapperTensorflow
+except Exception:
+    print("No mock frontend available")
+    mock_available = False
+
+# FrontEndManager shall be initialized and destroyed after all tests finished
+# This is because destroy of FrontEndManager will unload all plugins, no objects shall exist after this
+fem = FrontEndManager()
+ONNX_FRONTEND_NAME = "onnx"
```

Comment:
Why do we need these names?

Reply:
looks redundant, will be deleted

Reply:
ONNX_FRONTEND_NAME  has been deleted, other names are used in the code